examples/README.md

```diff
@@ -41,4 +41,5 @@ Overview
 | <a href="PUT a file to blobStorage account.xml">PUT a new file at your blobStorage account</a>                                                                                     | Use this policy to create a blobStorage account API abstraction and to create a new file at the configured container.                                                                                                                 |
 | <a href="DELETE a from to blobStorage account.xml">DELETE a file from your blobStorage account</a>                                                                                 | Use this policy to create a blobStorage account API abstraction and to delete a file at the configured container.                                                                                                                     |
 | <a href="GET a file from blobStorage account.xml">GET a file from your blobStorage account</a>                                                                                     | Use this policy to create a blobStorage account API abstraction and to retrieve a file from the configured container.                                                                                                                 |
-| <a href="Use custom error messages for jwt-validate policy with on-error handler.policy.xml">Use custom error messages for jwt-validate policy with on-error handler</a>                  | Provides the ability to use specific error messages when errors in the jwt-validate policy occur                                                                                                                                      |
\ No newline at end of file
+| <a href="Use custom error messages for jwt-validate policy with on-error handler.policy.xml">Use custom error messages for jwt-validate policy with on-error handler</a>           | Provides the ability to use specific error messages when errors in the jwt-validate policy occur                                                                                                                                      |
+| <a href="Show All Inbound Headers.policy.xml">Show All Inbound Headers</a>                                                                                                         | Shows all inbound headers and their values.                                                                                                                                                                                           |
```

Comment:
I meant that in the other examples only the first word starts with a capital letter and the rest is lowercase (unless it's an abbreviation or a name). Let's be consistent and rename the file from "Show All Inbound Headers" to "Show all inbound headers". Even better, we could use "list" instead of "show".